backend/tracim_backend/tests/utils.py

```diff
@@ -369,3 +372,16 @@ def create_1000px_png_test_image() -> BytesIO:
 
 TEST_CONFIG_FILE_PATH = os.environ.get("TEST_CONFIG_FILE_PATH")
 TEST_PUSHPIN_FILE_PATH = os.environ.get("TEST_PUSHPIN_FILE_PATH")
+
+
+@contextlib.contextmanager
+def tracim_plugin_loader(plugin_name, pluggy_manager, plugin_root_folder):
+    sys.path.append(plugin_root_folder)
+    plugin_module = importlib.import_module(plugin_name)
+    sys.path.remove(plugin_root_folder)
+    plugins = pluggy_manager.get_plugins()
+    plugin_module.register_tracim_plugin(pluggy_manager)
+    loaded_plugins = [p for p in pluggy_manager.get_plugins() if p not in plugins]
+    yield None
```

Comment:
What is the effect of this yield statement?

Reply:
It return a generator instead of the value. This is a specific mecanism for doing "lazy-loading" stuff.
In this particular case, it's used for a context manager, which mean that any code after the yield will be called at the end of the context. This is a nice pattern to ensure we do properly cleanup things.